content/en/_index.md

```diff
@@ -378,6 +438,22 @@ try (CloudClient dlp = CloudClient.create()) {
   // make a request with the client
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+
+# NOTE FOR SAMPLE AUTHOR: 
+# **DO NOT include this comment block in the sample.**
+# The comment below about clients applies to gRPC-based clients (clients with GAPIC_AUTO in .repo-metadata.json). For google-api-python-client, see https://googleapis.github.io/google-api-python-client/docs/thread_safety.html#the-httplib2http-objects-are-not-thread-safe. For other libraries, consult with
+# the library owner.
```

Comment:
@kurtisvg I tried to capture some of the nuance around thread-safety here, but it might make more sense to put it elsewhere.